Tests/StoreKitUnitTests/StoreKit2/StoreKit2ObserverModePurchaseDetectorTests.swift

```diff
@@ -34,6 +34,7 @@ class StoreKit2ObserverModePurchaseDetectorTests: StoreKitConfigTestCase {
     override func setUp() async throws {
         try AvailabilityChecks.iOS16APIAvailableOrSkipTest()
         deviceCache = .init()
+        try await super.setUp()
```

Comment:
Calling `super.setup()` here allows us to easily use the SKConfig file when running locally